### DISCUSSION TITLE
ctrlr.org/forums redirects to GitHub - where are all the past discussions?

### DISCUSSION THREAD

**Discussion by synthmutt** [RomanKubiak/ctrlr#682] [Q&A] [answered]
Recently ctrlr.org redirects to github.com/RomanKubiak/ctrlr. Where can all the past programming discussions be viewed now, which were available at ctrlr.org/forums ?

**Comment by damiensellier** [comment, marked as answer]
it's gone but you can use the waybackmachine :
https://web.archive.org/web/20240302141438/https://ctrlr.org/forums/

The programming section:
https://web.archive.org/web/20240213061140/https://ctrlr.org/forums/forum/general/programming/

**Reply by synthmutt** [reply]
With GitHub as the new home instead of ctrlr.org, I would have hoped discussions would somehow be transferred to GitHub, but at least this works

**Reply by damiensellier** [reply]
I know, unfortunately this is the only way to read from the old forum. Don't hesitate to ask question here if you need some help.
